Modules/Core/Mesh/include/itkBinaryMask3DMeshSource.h

```diff
@@ -205,12 +205,12 @@ class ITK_TEMPLATE_EXPORT BinaryMask3DMeshSource:public ImageToMeshFilter< TInpu
   IdentifierType **m_CurrentRow{nullptr};
   IdentifierType **m_CurrentFrame{nullptr};
 
-  unsigned short m_CurrentRowIndex{0};
-  unsigned short m_CurrentFrameIndex{0};
-  unsigned short m_LastRowNum{0};
-  unsigned short m_LastFrameNum{0};
-  unsigned short m_CurrentRowNum{200};
-  unsigned short m_CurrentFrameNum{2000};
+  int m_CurrentRowIndex{0};
```

Comment:
Why not use an unsigned type such as uint, ulong, size_t or `IdentifierType`?

Reply:
i guess for those (originally "unsigned short") variables the only correct type is "int", with "unsigned int" any test will crash (probably with "long" or "unsigned long" too). I don't remember exactly what i saw in debugger, can reproduce and post, some numeric-trait-whatever issue. But "unsigned short" is too small, test case in email can be reproduced (segfault).